ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -45,6 +45,9 @@ typedef NS_ENUM(NSInteger, ORKQuestionType) {
     /// The scale question type asks participants to place a mark at an appropriate position on a continuous or discrete line.
     ORKQuestionTypeScale,
     
+    /// The vertical scale question type asks participants to place a mark at an appropriate position on a continuous or discrete vertical line.
+    ORKQuestionTypeVerticalScale,
```

Comment:
Would prefer not to add this as a separate question type. See further comment below.